### ISSUE TITLE
The execution gets stuck at checkUpdateNeeded task on a Windows 7 PC with git SCM

### ISSUE THREAD

**Issue by Zlate87** [researchgate/gradle-release#81]
When I try to run on a Windows 7 PC with git SCM the release execution gets stuck at checkUpdateNeeded task.

When I try the same on a Mac, there is no issue.

Looking at the debug messages, the last entry is:

     >>> Running [git, --git-dir=MY_WORKSPACE.git, --work-tree=MY_WORKSPACE, remote, update]

If I try to run `git remote update` directly I'm prompted for username and password, so it looks that the issue is because git is waiting for username and password, but the plugin is not asking you for them.

**Comment by segfly** [comment]
For me (also on Windows 7 using cygwin) `git remote update` was also hanging. I was using an SSH remote and as it turns out, I added my private key to Eclipse, but it was not in my `~/home/<user>/.ssh` directory. Once I added this, I [checked my ssh access](https://help.github.com/articles/generating-ssh-keys/#step-4-test-everything-out) from the command line and found the release plugin stopped hanging.
